test/Microsoft.Extensions.ObjectPool.Test/Microsoft.Extensions.ObjectPool.Test.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
     <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(DeveloperBuild)' == 'true' ">$(DeveloperBuildTestTfms)</TargetFrameworks>
```

Comment:
@natemcmaster 

> As far as TFMs, my preference would be something like this 

```xml
<TargetFrameworks>netcoreapp2.1</TargetFrameworks>
<TargetFrameworks Condition="'$(DeveloperBuild)' != 'true' ">$(TargetFrameworks);netcoreapp2.0</TargetFrameworks>
<TargetFrameworks Condition="'$(OS)' == 'Windows_NT' AND '$(DeveloperBuild)' != 'true' ">$(TargetFrameworks);net461</TargetFrameworks>
```

Isn't this more "conditional code" that has the same effect?

Reply:
same effect, just my preference. I think I prefer it because it cascades adding the list of TFMs...just feels cleaner